slrealizer/realize_sdss.py

```diff
@@ -2,12 +2,11 @@
 from __future__ import division
 from __future__ import print_function
 
-from realize_sl import SLRealizer
-from utils.constants import *
-from utils.utils import *
+from slrealizer.realize_sl import SLRealizer
+from slrealizer.utils.constants import *
```

Comment:
In general we should only ever `import *` in an `__init__.py` file - everywhere else it makes the code less understandable (because we can't see where the different objects are defined). Better to do `import slrealizer.utils.constants as constants` and then live with variable names like `constants.G`.

Reply:
Done!